wp-job-manager-functions.php

```diff
@@ -1094,6 +1137,8 @@ function job_manager_dropdown_categories( $args = '' ) {
 		$r['pad_counts'] = true;
 	}
 
+	$r['search_category_slugs'] = wpjm_get_category_slugs_from_search_query_string();
```

Comment:
For consistency with how the other args are handled, I wonder if this should be wrapped with `if ( ! isset( $r['search_category_slugs'] ) ) { ... }`?

Reply:
I think what you mean is not have the `search_category_slugs` in the object when it's not necessary, right?

I really don't know if has this prop would be a problem, because if we don't have a search query string, it would be only a `[]`. But what we could do is change this line to something like:

```php
$search_category_slugs = wpjm_get_category_slugs_from_search_query_string();

if ( ! empty( $search_category_slugs ) ) {
  $r['search_category_slugs'] = $search_category_slugs
}
```

WDYT?